peregrine/tracking.py

```diff
@@ -163,19 +148,35 @@ def track(samples, channels,
 
   # Run tracking for each channel
   def do_channel(chan, n=None, q_progress=None):
-    loop_filter = loop_filter_class(*stage1_loop_filter_params)
     track_result = TrackResults(num_points, chan.prn)
 
     # Convert acquisition SNR to C/N0
     cn0_0 = 10 * np.log10(chan.snr)
     cn0_0 += 10 * np.log10(1000) # Channel bandwidth
-    cn0_est = swiftnav.track.CN0Estimator(1e3, cn0_0, 10, 1e3)
+    cn0_est = swiftnav.track.CN0Estimator(
+                                          bw=1e3,
+                                          cn0_0=cn0_0,
+                                          cutoff_freq=10,
+                                          loop_freq=1e3
+                                          )
 
     # Estimate initial code freq via aiding from acq carrier freq
     code_freq_init = (chan.carr_freq - IF) * \
                      gps_constants.chip_rate / gps_constants.l1
-    code_freq_init = 0
-    loop_filter.start(code_freq_init, chan.carr_freq - IF)
+    carr_freq_init = chan.carr_freq - IF
+    loop_filter = loop_filter_class(
+      loop_freq = stage1_loop_filter_params[2], \
```

Comment:
Arranging stage1_loop_filter_param as a dictionary would be more readable, I think.
It would look like this then:
loop_freq = stage1_loop_filter_params["loop_freq"]
etc